.github/workflows/release_bin.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: '^1.18'
+          go-version: '1.20'
```

Comment:
Let's add a comment here to reference the issue with 1.21, this way we can know in the future why it was pinned and if it's ok to change back

Reply:
Should we make this point to the version in the go.mod directly? So it always build with whatever is on there?

```
- uses: actions/setup-go@v4 with: go-version-file: 'path/to/go.mod'

```

Reply:
I have added both changes. Good finding @Itxaka.